frontend/src/employee-frontend/components/child-information/placements/PlacementRow.tsx

```diff
@@ -289,6 +291,12 @@ function PlacementRow({ placement, onRefreshNeeded, checkOverlaps }: Props) {
             </FixedSpaceRow>
           </ActionRow>
         )}
+
+        <RequireRole oneOf={['ADMIN']}>
+          <Gap size="s" />
+
+          <NewServiceNeeds serviceNeeds={placement.serviceNeeds} />
+        </RequireRole>
```

Comment:
I would hide this in production at least for now.

Reply:
done